src/tdd/isograma.ts

```diff
@@ -1,9 +1,18 @@
-function isograma(string) {
+type Vistos = {
+    [key: string]: boolean;
+};
+function isograma(s: string): boolean {
+    let letraComEsp = s.replaceAll("-", "");
+    letraComEsp = letraComEsp.replaceAll(" ", "");
 
-        if (string !== '-') {
-          callback(string);
-        }}
+    const letrasPalavras = "";
 
+    let olhados: Vistos = {};
+    for (let i = 0; i < letrasPalavras.length; i++) {
+        const letra = letrasPalavras[i];
+        olhados[letrasPalavras] = false;
+    }
```

Comment:
- A `length` dessa constante `letrasPalavras` é 0, já que ela é uma string vazia. O `for` loop deve passar por todas as letras da string `s` que é recebida como parâmetro da função.
- Lembre de tornar todas as letras da string `s` maíusculas ou minúsculas antes do `for` loop.
- Dentro do `for` loop, deve verificar se a `letra` já existe no objeto `olhados` (`if (olhados[letra] {...}`) e, caso ela não exista, ela deve ser colocada no objeto `olhados` como **true**.

Reply:
irei aplicar algumas alterações, obrigado!